src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -1,7 +0,0 @@
-// Elastic charts
-@import '@elastic/charts/dist/theme';
-@import '@elastic/eui/src/themes/charts/theme';
-
-// Public UI styles
-@import 'src/legacy/ui/public/index';
```

Comment:
Hm...cannot find this file. Does it exist?

Reply:
Hum, actually it does. `src/legacy/ui/public/_index.scss`. I thought these css files were no longer imported, but seeing https://github.com/elastic/kibana/pull/76085#discussion_r480156795, I might have been wrong

Reply:
After talking with Spencer and Tyler, this file is no longer used, so we should be good to remove it